backend/pkg/cbor/service.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file https://github.com/redpanda-data/redpanda/blob/dev/licenses/bsl.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// Package cbor provides the deserialization logic for CBOR encoded
+// payloads.
+package cbor
+
+import (
+	"regexp"
+
+	"github.com/redpanda-data/console/backend/pkg/config"
+)
+
+// Service represents messagepack cfg, topic name regexes.
+type Service struct {
```

Comment:
I think we don't need a service for this. Instead of creating a separate struct here I think we can handle this inside the serde directly no? The sole reason for this service seems to be for checking whether a topic is allowed or not.

Reply:
Yep, I can change that, but this is just the same way as it implemented for msgpack: https://github.com/redpanda-data/console/blob/master/backend/pkg/msgpack/service.go

Should msgpack be redesigned too?

Reply:
You don't need to change msgpack. The regexp config type was introduced after the msgpack PR was provided by another community user.

Reply:
Fixed